wordpress-mobile/WordPress-iOS
# Stats Revamp: Followers Donut Graph

This PR adds the Followers Donut Graph to the Total Followers Details Screen

Ref #18605

Implementation
<img width="300" alt="image" src="https://user-images.githubusercontent.com/88816724/174555172-e76dd54a-846e-4242-ac36-f8b748ab9bb2.png">

Design
<img width="362" alt="image" src="https://user-images.githubusercontent.com/88816724/174555295-09b5d6d1-caa2-4587-bc3f-74b20d178a32.png">



To test:
1. enable stats feature flags 
2. Go to Stats -> Insights.  Click Followers Totals
3. Followers Donut Graph should appear.  Try for a few sites with different Wordpress / email / publicise counts and the donut graph should perform what's mentioned here https://github.com/wordpress-mobile/WordPress-iOS/pull/18910#issuecomment-1163283723 

## Regression Notes
1. Potential unintended areas of impact
None

4. What I did to test those areas of impact (or what existing automated tests I relied on)
None

5. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
